dwst/scripts/lib/particles.js

```diff
@@ -120,18 +127,22 @@ function readInstructionName(parsee) {
     throw new InvalidParticles('an instruction name', String(parsee));
   }
   if (parsee.startsWith('}') || parsee.length === 0) {
-    throw new InvalidParticles(['('], String(parsee));
+    throw new InvalidParticles(quoteEach('('), String(parsee));
   }
   return instructionName;
 }
 
 function readInstructionArg(parsee) {
   const arg = parsee.readWhile(alphaChars.concat(digitChars));
-  if (parsee.startsWith('}') || arg.length === 0) {
+  if (parsee.startsWith('}')) {
+    throw new InvalidParticles(['an argument'].concat(quoteEach(')')), String(parsee));
```

Comment:
The line is a bit long which might justify splitting this into several consts. Other than that I'm fine with this.